plopfile.js

```diff
@@ -1,11 +1,52 @@
+import { execSync } from 'child_process'
+
 import { journeyPrompt } from './prompts/journey-prompt.js'
 import { questionKeyPrompt } from './prompts/question-key-prompt.js'
 import { questionPrompt } from './prompts/question-prompt.js'
 import { questionTypePrompt } from './prompts/question-type-prompt.js'
 import { sectionKeyPrompt } from './prompts/section-key-prompt.js'
 import { urlPathPrompt } from './prompts/url-path-prompt.js'
+import { radioChoicePrompt } from './prompts/radio-button-choices.js'
 
 export default function (plop) {
+  plop.setHelper('eq', (a, b) => {
+    return a === b
+  })
+
+  plop.setHelper('choicesObject', (choices) => {
+    const kebabCase = plop.getHelper('kebabCase')
+    const sentenceCase = plop.getHelper('sentenceCase')
+    const choicesArr = choices.split(',')
+
+    if (!choicesArr || !Array.isArray(choicesArr)) return '{}'
+
+    const choicesObj = {}
+    choicesArr.forEach((choice) => {
+      const key = kebabCase(choice.trim())
+      const label = sentenceCase(choice.trim())
+
+      choicesObj[key] = { label }
+    })
+
+    return JSON.stringify(choicesObj, null, 2)
+  })
+
+  plop.setHelper('choicesArray', (choices) => {
```

Comment:
Maybe call this something like "choiceToNextPageMappingArray"?  Or something that brings the next page concept into the name